kazoo/tests/test_connection.py

```diff
@@ -258,7 +258,7 @@ def back(state):
 class TestReadOnlyMode(KazooTestCase):
     def setUp(self):
         os.environ["ZOOKEEPER_LOCAL_SESSION_RO"] = "true"
-        self.setup_zookeeper(read_only=True)
+        self.setup_zookeeper()
```

Comment:
is `read_only=True` the default?

Reply:
No, this method takes some kwargs that will be passed to the kazoo client used in the test. This is not useful anymore, hence the removal